config/default/manager_disable_kubeadm.yaml

```diff
@@ -0,0 +1,5 @@
+- op: replace
+  path: "/spec/template/spec/containers/0/args"
+  value: 
+    - "--enable-leader-election"
+    - "--disable-kubeadm-bootstrapper=true"
```

Comment:
Is this patch now defaulting to true? If so, this could break the "battery included" case